wrapper/libvmaf.pc

```diff
@@ -5,7 +5,7 @@ includedir=/usr/local/include
 
 Name: libvmaf
 Description: Netflix's VMAF library
-Version: 1.3.14
+Version: 1.3.15
```

Comment:
1.4.0

Reply:
`CHANGELOG.md` and `VERSION` were addressed. But you missed the version in `wrapper/libvmaf.pc` and `wrapper/src/vmaf.cpp`.

Reply:
By the way, can't you place the version string in `wrapper/src/vmaf.cpp` at the top of the file, or place it in `wrapper/src/vmaf.h`. Placing a version string in the middle of the file looks weird to me, and could easily get missed when bump version.